cw-orch-environment/src/contract/paths.rs

```diff
@@ -153,12 +155,7 @@ mod artifacts_dir {
                         None
                     }
                 })
-                .ok_or_else(|| {
-                    CwOrchError::StdErr(format!(
-                        "Could not find wasm file with name {} in artifacts dir",
-                        name,
-                    ))
-                })?;
+                .ok_or_else(|| CwEnvError::WasmNotFound(name.to_owned()))?;
```

Comment:
It would be great if this error also included the path in which it was not found, as I've run into many a time where I wasn't sure if it was automatic or env or whatever.